test/elasticsearch.yml

```diff
@@ -0,0 +1,11 @@
+node:
```

Comment:
if we want that to be an end-to-end test, I'll think we'll use the `search()` method of fhirstore (hence elastic) to check that the record have been insterted. Do you think we should only check mongo ? I was asking myself if it would not be better to have a separated repo for e2e tests.